cmd/oceantv/broadcast_states.go

```diff
@@ -688,14 +688,19 @@ func onFailureClosure(ctx *broadcastContext, cfg *BroadcastConfig) func(err erro
 				_cfg.StartFailures++
 				// TODO: make this configurable in config.
 				const maxStartFailures = 3
-				if _cfg.StartFailures >= maxStartFailures {
+				if disableOnFirstFail || _cfg.StartFailures >= maxStartFailures {
 					_cfg.StartFailures = 0
 					_cfg.Enabled = false
 					notifier.Send(
 						context.Background(),
 						_cfg.SKey,
 						"health",
```

Comment:
Not for this PR, but let's start using different kinds of messages other than "health" for everything, so we have better notification granularity. All broadcast notification kinds can have the prefix "broadcast-". For example,this one could be "broadcast-hardware" since it is a hardware failure.

Reply:
We should probably define a "NotificationKind" enum so that it's clear what options there are.

Reply:
Good idea. I'll add this to the notify package.